_pages/policies.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: Policies
+subtitle: Policies
+permalink: /policies/
+---
+
+All StarCon attendees, speakers, volunteers, and sponsors are asked to abide by these policies.
+
+## Photography Policy
+
+All StarCon attendees, volunteers, speakers, and sponsors will be provided with a badge. Three colors of badges are provided.
+
+- Green means it is okay for photographers to take pictures of you.
+- Blue means photographers should ask permission.
+- Red means photos are not okay (they will not be published).
```

Comment:
I'd tweak this a bit:

- Green means it is okay for photographers to take pictures of you.
- Blue means photographers should ask permission before taking pictures of you.
- Red means photos should not be taken of you. (If any are taken by accident, they will not be published).

Note that "photographers" does not only apply to hired event photographers but also attendees taking pictures at the conference.

Reply:
I think we should use formal language here since it's a formal policy. i.e avoid use of 'okay' and other colloquialisms

Reply:
> Green means photographers have permission to take pictures of you

Reply:
Oh nvm I see you already had an alternate wording